docs/api/h.md

```diff
@@ -169,6 +169,15 @@ const pokedex = (pokemon) =>
 
 Props that represent event listeners, such as [`onclick`](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event), [`onchange`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event), [`oninput`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event), etc. are where you would assign [actions](../architecture/actions.md) to VNodes.
 
+Synthetic events can be added in the same way as long as their name starts with "on", so an event created with 
+```js
+const buildEvent = new Event('build')
```

Comment:
Double quotes for consistency, thanks.

Reply:
Sorry I don't understand, in the event name ? "build" ?

Reply:
done